valtech-sd/rule-harvester
# Add UDP Core Input

# Add UDP Core Input

## PR Details

- Add Core UDP Input provider
- Add example in examples directory to demonstrate how to use the input provider.
- This is a listener only and does not provide functionality to transmit UDP packets. 
- Some small changes were made to amqp examples to remove dependency on amqp secrets for non amqp examples.

## How to test

1. Git checkout project and open a terminal to the root folder
2. Run `npm run build`
3. run `cd ./examples`
4. Run `npm run start-udp`
5. Open a new terminal and run ` cat ./example_ca_order.json | nc -4u -w1 localhost 3333`
6. A new file should have appeared in `./examples/output_order_dispatch` with the order results
